amp/TEMPLATE/reampv2/src/modules/geocoder/components/layout/table/ActivityTable.jsx

```diff
@@ -1,35 +1,40 @@
-import React, {Component} from 'react';
+import React, {Component, useState} from 'react';
 import BootstrapTable from "react-bootstrap-table-next";
 import paginationFactory from 'react-bootstrap-table2-paginator';
 import {bindActionCreators} from "redux";
 import {connect} from "react-redux";
-import TableActions from "./ActivityActions";
 
 import './table.css';
-import GeocodedActivity from "./LocationsTabel";
+import {
+    loadActivities,
+    selectActivityForGeocoding,
+    selectAllActivitiesForGeocoding
+} from "../../../actions/activitiesAction";
+import {Loading} from "../../../../../utils/components/Loading";
+
+const SelectedActivitiesMessage = (props) =>  {
+    return (
```

Comment:
when using functional components its better to do ({size}) instead of props.size so its clear what we expect being passed as props and the code is simple to read